src/documentation/literal.rs

```diff
@@ -14,8 +14,8 @@ pub enum CommentVariant {
     TripleSlash,
     /// `//!`
     DoubleSlashEM,
-    /// `#[doc=`
-    MacroDocEq,
+    /// `#[doc=` with the length of possible `r#`s
```

Comment:
Is this the length of `r###` with or without `r` or just the number `#`?

Reply:
it's both, added a note to the comment